docs/en/mutations.md

```diff
@@ -138,8 +138,70 @@ Now imagine we are debugging the app and looking at the devtool's mutation logs.
 
 ### Committing Mutations in Components
 
-You can commit mutations in components with `this.$store.commit('xxx')`, or use the `mapMutations` helper which maps component methods to `store.commit` calls (requires root `store` injection):
+Vuex mutations are not directly available as callable methods inside your Vue templates. To trigger mutations from your templates, expose them via your methods.
 
+One way to do this is to trigger regular Vue methods that commit Vuex mutations.  The Vue method can call `this.$store.commit('xxx')`:
+
+``` html
+<button @click="incrementMethod">Increment</button>
+```
+``` js
+export default {
+  // ...
+  methods: {    
+    incrementMethod(){
```

Comment:
I think we should define this method as `increment` if we rewrite it with `mapMutations(['increment'])` later for consistency.